lib/api_entity.rb

```diff
@@ -72,11 +81,9 @@ def initialize(attributes = {})
   def attributes=(attributes = {})
     @attributes = attributes
 
-    if attributes.present?
-      attributes.each do |name, value|
-        if respond_to?(:"#{name}=")
-          send(:"#{name}=", value.is_a?(String) && value == 'null' ? nil : value)
```

Comment:
Are we sure this type cast hack isn't needed anywhere? - does look wrong but presumably was added at some point for good reason

Reply:
I've never seen a value as the literal string 'null'. The ruby json libs would serialize a ruby `nil` as a json `null`  and I couldn't see anything in the serializers/models/presenters that might be doing this

Reply:
It was added around the display of synonyms :eyes: https://github.com/trade-tariff/trade-tariff-frontend/commit/b48c050c376ad92b0b6e6fab443b9e9f0ad0950b

Reply:
Yeah, this synonym code (really search references) isn't implemented in the frontend anymore and we're never going to be serializing the literal string 'null'. This could be a weird bug in the JSONAPI gem from 2012, maybe but this isn't possible AFAIK

Reply:
Yeh, I'd say at the time they were hitting a bug in their serializer - that commits 10 years old, should be fine :+1: